tests/Fixer/Basic/Psr4FixerTest.php

```diff
@@ -47,7 +47,7 @@ class bar {}
 
         $expected = <<<'EOF'
 <?php
-class Bar {}
+class Psr4_Foo_Bar {}
```

Comment:
This is a revert of a change from https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5348#discussion_r539700027

Reply:
I don't think PSR4 permits having a class name like that. Only PSR0?

Reply:
I guess it should not touch the original name `Psr4_fOo_bAr` at all right? So neither the previous or the new version is correct.

The more I'm thinking (reading/writing code) for the PSR0/4 fixer the more I think the common fixer was a bad idea.

PSR-0 fixer could stay with the `dir` config value and PSR-4 one could some config to fully support PSR-4 which is changing the whole namespace to the expected one, not only namespace suffix.

Reply:
Original spec of this fixer was to keep `Psr4_Foo_Bar` here - ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.11.2/tests/Fixer/Basic/Psr4FixerTest.php#L50
I would indeed keep the original version here.

We aim to fix as much, without much of code breaking.